.github/workflows/lint-test.yaml

```diff
@@ -97,13 +97,13 @@ jobs:
       - name: Migrations and run tests with coverage.py
         run: |
           python manage.py makemigrations --check
-          python manage.py migrate
           coverage run manage.py test --no-input
           coverage report -m
         env:
           CI: True
-          DATABASE_URL: postgres://pysite:pysite@localhost:7777/pysite
-          METRICITY_DB_URL: postgres://pysite:pysite@localhost:7777/metricity
+          DATABASE_URL: postgres://pysite:pysite@localhost:7778/pysite
+          TEST_DB_URL: postgres://pysite:pysite@localhost:7778/pysite
```

Comment:
What's the point of separate URLs if they're identical anyway? Do we _need_ the flexibility to make them different if we want?

Reply:
They're both needed as one is used in the migrations, and one is used in the tests. Making them one environment variable would be annoying as locally and in the compose they aren't going to refer to the same thing.

Reply:
Oh sorry, I see that they are different by default in docker-compose.yml. I confused this file with docker-compose.yml.